src/ontology/mondo-edit.obo

```diff
@@ -533162,6 +533196,7 @@ synonym: "intellectual developmental disorder, autosomal dominant 57" EXACT [OMI
 synonym: "intellectual disability, autosomal dominant 57" EXACT []
 synonym: "mental retardation, autosomal dominant 57" RELATED DEPRECATED []
 synonym: "MRD57" RELATED DEPRECATED []
+synonym: "TLK2-Related Neurodevelopmental Disorder" EXACT []
```

Comment:
added without x-ref; same synonym as the one below but with different capitalization